.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: I want help writing my application from Flutter's community on Reddit
```

Comment:
Can we only add the link to stackoverflow? Too many choices may make it difficult to to chose one.

Reply:
sounds good to me, I've included both Reddit and SO following this [comment](https://github.com/flutter/flutter/issues/61094#issuecomment-668777210)